packages/terra-spacer/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-spacer",
+  "main": "lib/Spacer.js",
+  "version": "0.0.0",
+  "description": "This component is used to provide margin and/or padding space between two components based on the given values.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-spacer",
+    "Spacer",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.6.1"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-bps": "0.0.4",
+    "terra-base": "^2.6.1",
+    "terra-toolkit": "^2.1.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
```

Comment:
Update to `"compile:clean": "rimraf lib",`

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/0b85f9f949ade6f12fb384a5fff8facfaa91784b#diff-f3bfa5e0bf12fb4af2128aaf883cc625R37